Maestro/subscriptions.json

```diff
@@ -653,6 +653,23 @@
         }
       }
     },
+    // Automate merging runtime release/6.0 branch into release/6.0-maui
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/runtime/blob/release/6.0/**/*"
```

Comment:
You want an extra slash here `https://github.com/dotnet/runtime/blob/release/6.0//**/* otherwise it will pick up any 6.0* branch.

Reply:
@mmitche thanks, updated.

Reply:
Looks like I don't have permissions to merge